tests/iohandling/test_parsing.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+from unittest.mock import MagicMock, patch
+import sys
+
+from fw_pipeline_io.iohandling import parsing
+
+
+class TestParsing(unittest.TestCase):
```

Comment:
As above, consider adding brief, one-sentence doc strings for each test to state what is being checked.

Reply:
added